tests/integ/test_integration.py

```diff
@@ -302,6 +302,63 @@ def test_list_files_range_cc_asn(client):
     assert len(results) == 7
 
 
+# # get_measurement_meta # #
+
```

Comment:
I would also add a test case for the scenario of a measurement which doesn't have an input.

Reply:
Yes - added support in the last commit.